neo/bin/prompt.py

```diff
@@ -104,7 +104,7 @@ class PromptInterface(object):
                 'wallet tkn_approve {token symbol} {address_from} {address to} {amount}',
                 'wallet tkn_allowance {token symbol} {address_from} {address to}',
                 'wallet tkn_mint {token symbol} {mint_to_addr} (--attach-neo={amount}, --attach-gas={amount})',
-                'wallet tkn_register {addr} ({addr}...)',
+                'wallet tkn_register {addr} ({addr}...) (--from-addr={addr})',
```

Comment:
I do wonder if we even want to support `tkn_register` as a native command at all. It's not part of NEP-5, and there are conversations around changing the de facto standard (e.g. camel case instead of underscores). It's easy enough to do with `testinvoke`, so we may just want to get rid of support for it entirely?

Reply:
I think its good to keep around.  We're working on an NEP for sale related ICO methods, so once that is complete we can change the syntax to camelCase or whatever it ends up being.